tests/test_flows.py

```diff
@@ -692,6 +692,18 @@ def my_flow():
         state = my_flow._run()
         assert state.is_completed()
 
+    def test_user_timeout_is_not_hidden(self):
+        @flow(timeout_seconds=30)
+        def my_flow():
+            raise TimeoutError("Oh no!")
+
+        state = my_flow._run()
```

Comment:
Should we remove `_run` and switch the tests to use `return_state=True` where needed?

Reply:
Oh... I didn't know that was implemented while I was gone. I guess so.